### PR TITLE
Add back orphaned file

### PR DESCRIPTION
**Pull request by Loquacity**

### CODE REVIEW

File: modules/salt/pages/salt-states.adoc [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+* Minion2 part of [Org1, Group1] wants vim version 2 gets version 2 (Group Level)
+* Minion3 part of [Org1, Group1] wants any version, gets version 2 (Org Level)
+* Minion4 part of[Org1, Group2] wants any version, gets vim version 1 (Org Level)
+
+
+[[bp.chapt.config.mgr.salt.states.storage.locations]]
+== Salt States Storage Locations
+
+
+The {productname} salt-master reads its state data from three file root locations.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
The {productname} salt-master reads its state data from three directories.
```

File: modules/salt/pages/salt-states.adoc [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+The directory [path]``/usr/share/susemanager/salt``
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
The directory [path]``/usr/share/susemanager/salt`` is used by {productname} and comes from the susemanager-sls.
```

File: modules/salt/pages/salt-states.adoc [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+ is used by {productname} and comes from the susemanager-sls.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
```

File: modules/salt/pages/salt-states.adoc [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+It is shipped and updated together with {productname} and includes certificate setup and common state logic to be applied to packages and channels.
+
+The directory [path]``/srv/susemanager/salt`` is generated by {productname} and based on assigned channels and packages for minions, groups and organizations.
+This file will be overwritten and regenerated.
+This could be thought of as the {productname} database translated into salt directives.
+
+The third directory [path]``/srv/salt`` is for custom state data, modules etc.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
The third directory [path]``/srv/salt`` is for custom state data, modules, etc.
```

File: modules/salt/pages/salt-states.adoc (hunk 2 above)
Comment (keichwa) [reply modules/salt/pages/salt-states.adoc]: what's susemanager-sls?  Is this a package name?  Either some kind of markup is missing or spelling has to be changed.

File: modules/salt/pages/salt-states.adoc [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+{productname} does not operate within or utilize this directory.
+However the state data placed here affects the Highstate of minions and is merged with the total state result generated by {productname}.
+
+[[bp.chapt.config.mgr.susemgr.states]]
+== {productname} States
+
+
+All sls files created by users will be saved to disk on the salt-master server.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
All user created SLS files will be saved to disk on the salt-master server.
```
Do we actually use salt-master?  Or Salt master.
I vote to cap SLS as you usually write PNG files, etc.

File: modules/salt/pages/salt-states.adoc [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+These files will be placed in [path]``/srv/susemanager/salt/``
+ and each organization will be placed within its own directory.
+Although these states are custom, these states are created using {productname}
+.
+The following provides an overview of directory structure:
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
The following provides an overview of the directory structure:
```

File: modules/salt/pages/salt-states.adoc [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+│   │    ... other files needed by states ...
+│   └── motd.sls     # user created
+            ... other sls files ...
+----
+
+[[bp.chapt.config.mgr.pillar.data]]
+== Pillar Data
+
+
+SUSE Manager exposes a small amount of internal data as Pillars which can be used with custom {sls} states.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]: is {sls} the right macro here?

File: modules/salt/pages/salt-states.adoc [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+Data that is exposed includes group membership, organization membership, and file roots.
+These are managed either automatically by {productname}, or manually by the user.
+
+To avoid hard-coding organization IDs within {sls} files, a pillar entry is added for each organization:
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]: Capitalizing pillar or not?

File: modules/salt/pages/salt-states.adoc [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+    - mode: 644
+----
+
+
+
+[[bp.sect.groupstates]]
+== Group States
+
+Pillar data can be used to perform bulk actions, like applying all assigned states to minions within the group.
+This section contains some example of bulk actions that you can take using group states.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]: execute?  Later we say perform (what I'd prefer here as well)

File: modules/salt/pages/salt-states.adoc [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+
+Apply the highstate to all minions in the group:
+
+----
+salt -I 'group_ids:GID' state.apply
+----
+
+
+[[salt.pillars.setpackageendpoint]]
+== Use pillars to set the package download endpoint
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
== Use Pillars to Set the Package Download Endpoint
```

File: modules/salt/pages/salt-states.adoc [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+ is used by {productname} and comes from the susemanager-sls.
+It is shipped and updated together with {productname} and includes certificate setup and common state logic to be applied to packages and channels.
+
+The directory [path]``/srv/susemanager/salt`` is generated by {productname} and based on assigned channels and packages for minions, groups and organizations.
+This file will be overwritten and regenerated.
+This could be thought of as the {productname} database translated into salt directives.
+
+The third directory [path]``/srv/salt`` is for custom state data, modules etc.
+{productname} does not operate within or utilize this directory.
+However the state data placed here affects the Highstate of minions and is merged with the total state result generated by {productname}.
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
However, the state data placed here affects the Highstate of minions and is merged with the total state result generated by {productname}.
```
see the other however sentence later on.

File: modules/salt/pages/salt-states.adoc [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+
+
+[[salt.pillars.setpackageendpoint]]
+== Use pillars to set the package download endpoint
+
+By default, {productname} assumes that the download endpoint to use is the FQDN of the {productname} server, or the {productname} Proxy.
+However, there are some cases where you might like to use a different FQDN as the download endpoint.
+The most common example is if you need to use load balancing, caching proxies, or in environments with complicated networking requirements.
+
+To change the package download endpoint, you can manually adjust three salt pillars:
```
Comment (keichwa) [inline modules/salt/pages/salt-states.adoc]:
```suggestion
To change the package download endpoint, you can manually adjust three Salt pillars:
```

**Comment by keichwa** [comment]
Abid Mehmood <notifications@github.com> writes:

> admd approved this pull request.
>
> I didn't go through in detail but it looks good. I did assume though
> that this text has already been reviewed once before.
> Thank you for the quick action on this missing content in the
> documentation :+1:

It has, but probably in a hurry ;)

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

File: modules/salt/pages/salt-states.adoc (hunk 1 above)
Comment (keichwa) [reply modules/salt/pages/salt-states.adoc]: maybe, file root is okay.  It is also used later on.

File: modules/salt/pages/salt-states.adoc (hunk 8 above)
Comment (Loquacity) [reply modules/salt/pages/salt-states.adoc]: I don't think we've capped it elsewhere.

File: modules/salt/pages/salt-states.adoc (hunk 9 above)
Comment (Loquacity) [reply modules/salt/pages/salt-states.adoc]: Dunno. I think the whole thing needs a proper edit, it's old content.

**Comment by Loquacity** [comment]
> I didn't go through in detail but it looks good. I did assume though that this text has already been reviewed once before.
> Thank you for the quick action on this missing content in the documentation

It is old content, and it definitely needs a proofread and update, but the priority for the moment is to get it in. I'm glad @keichwa took the time to pick up the most egregious errors, though!

File: modules/salt/pages/salt-states.adoc (hunk 7 above)
Comment (Loquacity) [reply modules/salt/pages/salt-states.adoc]: Looks out of place, I think we can probably delete it entirely.

```suggestion
SUSE Manager exposes a small amount of internal data as Pillars which can be used with custom states.
```

**Pull request merged**